codebuild/codebuild_project_container_build_docker_hub/code_build_terraform.tf

```diff
@@ -38,17 +38,24 @@ resource "aws_codebuild_project" "code_pipeline_container_build_docker_hub" {
     environment_variable {
       name  = "DOCKERHUB_USERNAME"
       value = var.docker_hub_username
+      type  = SECRETS_MANAGER
```

Comment:
What are you going to pass into these vars? The value in secrets manager is JSON with a username and password property. I think if you pass in a secrets manager secret you have to then pass the path to the value you want to pull out.

Reply:
I see, this is already being parsed out from the calling module. I've added another commit to remove this for now. Would adding a `registry_credential` resolve this problem ?

Reply:
Still not having much joy with this (see latest commit) maybe it's the jsondecode function, the secrets are still output in plaintext when running a `terraform plan`.